FrontEnd/src/components/basicPage/BasicPage.jsx

```diff
@@ -30,6 +32,23 @@ function BasicPage() {
   const auth = useAuth();
   const user = useUser();
   const userData = user.user;
+
+  const StyledToastContainer = styled(ToastContainer)`
+  &&& .Toastify__toast-container {
+    padding: 0;
+  }
+  .Toastify__toast {
+    font-size: 14px;
+    border-radius: 2px;
+    box-shadow: none;
+    border: 1px solid var(--Dust-Red-2, #FFCCC7);
+    padding: 9px 16px;
+    gap: 10px;
+  }
+  .Toastify__toast-body {
+    padding: 0;
+  }`;
```

Comment:
I would recommend including these styles as simple CSS. No need to add a new package and increase the bundle size (Unpacked Size: 1.65 MB)

Reply:
I haven't find possibility to add this styles as simple css. That's why I've implemented it this way (as documentation recommends).
https://fkhadra.github.io/react-toastify/how-to-style/

Reply:
I've styled it in a little bit easier way, I suppose. 
Is it okay?